tests/tests_pytorch/strategies/test_ddp.py

```diff
@@ -18,42 +18,37 @@
 from torch.nn.parallel.distributed import DistributedDataParallel
 
 import pytorch_lightning as pl
-from pytorch_lightning import seed_everything, Trainer
+from pytorch_lightning import Trainer
 from pytorch_lightning.callbacks import Callback
 from pytorch_lightning.demos.boring_classes import BoringModel
 from pytorch_lightning.strategies import DDPStrategy
-from tests_pytorch.helpers.datamodules import ClassifDataModule
 from tests_pytorch.helpers.runif import RunIf
-from tests_pytorch.helpers.simple_models import ClassificationModel
 
 
 @RunIf(min_cuda_gpus=2, standalone=True)
 def test_multi_gpu_model_ddp_fit_only(tmpdir):
-    dm = ClassifDataModule()
-    model = ClassificationModel()
-    trainer = Trainer(default_root_dir=tmpdir, max_epochs=1, accelerator="gpu", devices=2, strategy="ddp")
-    trainer.fit(model, datamodule=dm)
+    model = BoringModel()
```

Comment:
Proposal unrelated to the sklearn change or the standalone failures: Let's just delete these tests? They basically test nothing other than that DDP can launch.

cc @awaelchli

Reply:
They probably can be replaced with a simpler launcher test. But they are here for a reason, from a time when Lightning was borked when calling multiple trainer.x methods in a row. They act as a regression test in that sense.

Reply:
yes, I was thinking too to remove them, but they fail on the ci, so was afraid to just remove them directly.